{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/settings.py

```diff
@@ -40,6 +40,8 @@
 INSTALLED_APPS = [
     'django.contrib.admin',
     'django.contrib.auth',
+    'rest_auth',
+    'rest_auth.registration',
```

Comment:
These rest auth one should go on third_party_apps one.

Reply:
Oops, that's where it was supposed to be. I accidentally pasted it into the wrong section. Fixed.